publication-doi-commons/src/main/java/no/unit/nva/immutables/Tupled.java

```diff
@@ -0,0 +1,16 @@
+package no.unit.nva.immutables;
+
+import org.immutables.value.Value;
+
+/**
+ * Tupled annotation will be used to generate simple tuples in reverse-style, having construction methods of all
+ * annotations.
+ */
+@Value.Style(// Tupled annotation will serve as both immutable and meta-annotated style annotation
+    typeAbstract = "*Def",
+    typeImmutable = "*",
+    allParameters = true, // all attributes will become constructor parameters
+    // as if they are annotated with @Value.Parameter
+    visibility = Value.Style.ImplementationVisibility.PUBLIC, // Generated class will be always public
+    defaults = @Value.Immutable(builder = false)) // Disable copy methods and builder
+public @interface Tupled {}
```

Comment:
```
// Look, custom annotation instead of @Value.Immutable
// and the style is also attached!
@Tupled interface RgbDef {
  double red();
  double green();
  double blue();
}
// ...
Rgb color = Rgb.of(0.4, 0.3, 1.0);
````